src/var/timeoutSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global setTimeout: false */
+	return window.setTimeout || setTimeout;
```

Comment:
This is a long comment, see https://github.com/mzgol/jquery/blob/7eef60f491be191e5844f3a3420bee238c7d73f3/src/intro.js#L19-L25. Should I repeat this comment in every of those 4 files?